### PR TITLE
updating jenkins workers

### PR DESCRIPTION
**Pull request by JoeNunnelley**
- relaxing chef version restriction to use latest 12.x version
- moving calls to ::docker out of common
- adding additional init q command for ssh service

### CODE REVIEW

**Comment by JoeNunnelley** [comment]
retest this please

File: recipes/_common.rb [hunk 1]
```diff
@@ -63,6 +61,7 @@
 bash 'fix ssh' do
   code <<-EOH
     sed -i 's/MACs /MACs hmac-sha1-96,/' /etc/ssh/sshd_config
+    sudo init q
```
Comment (michaelb990) [inline recipes/_common.rb]: what does this do?

Comment (JoeNunnelley) [reply recipes/_common.rb]: init q initializes the services list. I'm seeing issues with ssh and docker services not being properly initialized by their cookbooks. Running init q fixes this initialization problem.   This is a temporary fix while I look for a more correct solution.

**Comment by JoeNunnelley** [comment]
retest this please

**Comment by JoeNunnelley** [comment]
merging this as I need it merged to verify the packer piece of the process.

**Pull request merged**